requirements/requirements.linux-ubuntu.txt

```
@@ -11,7 +11,7 @@ bleach==6.0.0
 cffi==1.15.1
 comm==0.1.2
 contourpy==1.0.7
-cryptography==39.0.2
+cryptography==41.0.0
```

use latest version instead
```suggestion
cryptography==41.0.1
```